### DISCUSSION TITLE
Naming the Wiser Newsletter

### DISCUSSION THREAD

**Discussion by nathenf** [life-itself/community#188] [General]
We are well underway with planning for the #166 - Wiser Newsletter and here we can have an open discussion about the naming of the newsletter. 

It's goal is to be the signal through the noise when is comes to responses to the multiple crises we face.

So far my ideas:

- The Polycrisis Dispatch
   - Dispatch is often used to define a regular send out (either newspaper, or other media), yet the true definition is to **send off or away with promptness or speed**, I like this neat play on words as we also wish the send away the polycrisis (this is simplistic but there is some merit I think)
   - Polycrisis is a new term (@rufuspollock: my 2c: for me polycrisis is still pretty niche as a term), however it is one of the only words that really defines what we are up against currently. It is also a word that doesn't require much explaining (unlike metamodernism) 

We can just go for Wiser newsletter, as well. 

Curious to hear your thoughts and ideas 

@rufuspollock @shiweiart @theo-cox @catherinet1 @EilidhRoss1

**Comment by catherinet1** [comment]
I don't resonate with the 'wish to send away the polycrisis'... for me, the metaphor might be more about navigating the polycrisis. And so 'The Polycrisis Dispatch' doesn't land with me with wordplay. In 3 words, the flavour of that name to me is: intellectual; old-fashioned (like an old newspaper) but maybe in a quirky, Wes Anderson's _The French Dispatch_  kind of way; exclusive. And you're right, there is a kind of haste to it -- which as I think about it maybe sits at odds with a kind of patience or long/deep-time-ness that might be associated with wisdom. They're really two quite differently-vibed name options.

The flavour of 'Wiser newsletter' to me is more bland, flat, and open/inclusive.

Agree that 'polycrisis' also seems to me more accessible than 'metamodernism' and that yes, it is still not common parlance.

Do we have a finalised brand narrative and attributes yet? I wanted to look to those to see how the sound and feel of this name fits or aligns with that. 

I feel a bit out of the loop on the Wiser work to brainstorm some ideas rn, but could catch up and do some brainstorming if that would be helpful. Otherwise also just happy to feedback on ideas in whatever ways are supportive, i trust you'll let me know what you find helpful @nathenf.

**Comment by catherinet1** [comment]
Also, from 'The Polycrisis Dispatch', I expect a roundup of (news) reports on the polycrisis. i.e. focus on the crisis as opposed to the responses and opportunities for positive collaboration, engagement, etc. , whereas my understanding of this newsletter is that it would be more about Wise(r) Responses to Polycrisis?

**Comment by nathenf** [comment]
Thanks @catherinet1 for your feedback! We are definitely getting some more clarity through the branding process and will update again soon.
